cdap-docs/reference-manual/source/http-restful-api/http-restful-api-v3/metrics.rst

```diff
@@ -98,114 +373,162 @@ The general form of a metrics request is::
-run of a flow, you can issue an HTTP GET method::
+  POST <base-url>/metrics/query?context=namespace.default.app.CountRandom.flow.CountRandom.run.
+    bca50436-9650-448e-9ab1-f1d186eb2285.flowlet.splitter&metric=system.process.events.processed&aggregate=true
 
-  GET <base-url>/metrics/system/apps/CountRandom/flows/CountRandomFlow/runs/13ac3a50-a435-49c8-a752-83b3c1e1b9a8/flowlets/
-          splitter/process.events.processed?aggregate=true
+which will return something similar to::
+
+  {"startTime":0,"endTime":0,"series":[{"metricName":"system.process.events.processed",
+   "grouping":{},"data":[{"time":0,"value":11188}]}]}
 
 If the run-ID is not specified, we aggregate the events processed for all the runs of this flow.
 
 If you want the number of input objects processed across all Flowlets of a Flow, you address the metrics
 API at the Flow context::
 
-  GET <base-url>/metrics/system/apps/CountRandom/flows/
-    CountRandomFlow/process.events.processed?start=now-5s&count=5
+  POST <base-url>/metrics/query?context=namespace.default.app.CountRandom.flow.CountRandom.flowlet.*
+    &metric=system.process.events.processed&start=now-5s&count=5
+
+Similarly, you can address the context of all Flows of an Application, an entire Application, or the entire 
+namespace of a CDAP instance::
+
+  POST <base-url>/metrics/query?context=namespace.default.app.CountRandom.flow.*
+    &metric=system.process.events.processed&start=now-5s&count=5
 
-Similarly, you can address the context of all flows of an Application, an entire Application, or the entire CDAP::
+  POST <base-url>/metrics/query?context=namespace.default.app.CountRandom
+    &metric=system.process.events.processed&start=now-5s&count=5
 
-  GET <base-url>/metrics/system/apps/CountRandom/
-    flows/process.events.processed?start=now-5s&count=5
-  GET <base-url>/metrics/system/apps/CountRandom/
-    process.events.processed?start=now-5s&count=5
-  GET <base-url>/metrics/system/process.events?start=now-5s&count=5
+  POST <base-url>/metrics/query?context=namespace.default
+    &metric=system.process.events.processed&start=now-5s&count=5
 
 To request user-defined metrics instead of system metrics, specify ``user`` instead of ``cdap`` in the URL
 and specify the user-defined metric at the end of the request.
 
 For example, to request a user-defined metric for the *HelloWorld* Application's *WhoFlow* Flow::
 
-  GET <base-url>/metrics/user/apps/HelloWorld/flows/
-    WhoFlow/flowlets/saver/names.bytes?aggregate=true
+  POST <base-url>/metrics/query?context=namespace.default.app.HelloWorld.flow.WhoFlow.flowlet.saver
+    &metric=user.names.bytes&aggregate=true
 
-To retrieve multiple metrics at once, instead of a GET, issue an HTTP POST, with a JSON list as the request body that enumerates the name and attributes for each metrics. For example::
+Retrieving multiple metrics at once, by issuing an HTTP POST request with a JSON list as
```

Comment:
I think so, since it was a feature in v2 that is currently no longer present. If anyone had used and was looking for it...in fact I will add it as a known issue.